### ISSUE TITLE
Tests using proxyquire failing when run through wallaby

### ISSUE THREAD

**Issue by kennethlynne** [wallabyjs/public#727]
Running this in the command line works `ava app/compiled/tests/unit/tests/**/*.js app/compiled/src/**/*.spec.js` but with the 

I have tried setting workers to 1, and `recycle: true` without seeing any different behaviour.

![image](https://cloud.githubusercontent.com/assets/390817/17401394/b6a1494c-5a4d-11e6-9dc3-6b889f09f5f1.png)

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {
  return {
    files: [
      'app/src/**/*.ts',
      {pattern: 'app/tests/**/*', instrument: false, load: true, ignore: false},
      '!app/src/**/*.spec.ts',
      '!app/tests/unit/tests/**/*'
    ],

    tests: [
      'app/src/**/*.spec.ts',
      'app/tests/unit/tests/**/*.ts'
    ],

    compilers: {
      '**/*.ts': wallaby.compilers.typeScript()
    },

    env: {
      type: 'node'
    },

    testFramework: 'ava'
  };
};
```

### Code editor or IDE name and version
WebStorm v11

### OS name and version
OSX

**Comment by kennethlynne** [comment]
Changing 

![image](https://cloud.githubusercontent.com/assets/390817/17418174/5de2982a-5a97-11e6-9ff0-27410479b9d8.png)

To 

![image](https://cloud.githubusercontent.com/assets/390817/17418181/69c38c12-5a97-11e6-812d-fbe5f5ed3c41.png)

weirdly had a huge impact on "hot reload" issues (tests not re-running on change)

**Comment by ArtemGovorov** [comment]
Hmm, interesting. Could you please provide a sample repo where I could reproduce it?

**Comment by kennethlynne** [comment]
Seems like some cleaning up from our side + the fix for #728 fixed this! Thanks for the fast follow up and fix!
